### DISCUSSION TITLE
✅ How to switch to active window from main (servers) screen

### DISCUSSION THREAD

**Discussion by freepaddler** [isontheline/pro.webssh.net#544] [Q&A] [answered]
Hello, just started using webssh and found one problem. May be i'm too stupid, but i did not find the way to switch to existing active connections or to the window list from main (servers) screen

to switch to existing connection:
+ tap any connection and from there go to the window list. but this always creates new connection, even if i'm already connected to
+ tap mashREPL, with the same effect - always new mashREPL window

I'm really disheartened about it. May be there is a way to get active window list from "servers" screen?

**Comment by isontheline** [comment, marked as answer]
Hello @freepaddler 👋

Have you tried this howto?
https://webssh.net/documentation/help/howtos/launching-multiple-terminals/

When enabled you will able to switch to other windows.
Just don't exit the connections, just switch to other windows.

**Reply by freepaddler** [reply]
oh, thanks alot! this is (i suppose) the only place i didn't touch :)))

**Reply by isontheline** [reply]
You're Welcome
